### ISSUE TITLE
[Quest] The Last Line Of Defense - Icecrown ID: 13086

### ISSUE THREAD

**Issue by gonimajuan** [WoWManiaUK/Blackwing-Lair#4589]
**Links:**

https://www.wowhead.com/quest=13086/the-last-line-of-defense

from WoWHead or our Armory

**What is happening:** The Frostbrood Destroyers are not dying to the cannon shots.

**What should happen:** You should use the cannons atop of the Argent Vanguard towers to kill the Frostbrood Destroyers

**Is it crashing the server?:** No

**Other Information:** The cannons seem to deal low damage to the Destroyers, who after receiving just a small amount of damage, regenerate back to full health

**Comment by Echinaceax** [comment]
I have a question @gonimajuan :)

Do you actually "see" the cannons ?
Reason i ask is because i cant without the gm on in PTR

@devs
I dont see the cannons without my gm on (my phasing i think) but anyway
tested and https://www.wowhead.com/npc=30575/frostbrood-destroyer are not taking damage 

- not all the mobs take damage as far as i can see 

![some do](https://user-images.githubusercontent.com/39439201/86218771-926cca80-bb81-11ea-996c-d154788870be.jpg)

**Comment by gonimajuan** [comment]
@Echinaceax Yes, i do see the cannons. 

Not all the enemies take damage when they're too far away

**Comment by Echinaceax** [comment]
yes i saw that too :)
Thanks @gonimajuan

**Comment by jgilbert201** [comment]
Echi, the reason you won't be able to see them is due to phasing. it's part of the quest line. Confirmed this still doesn't work due to low damage after following quest line.

The only way to kill these, is to aggro one whilst flying, then jump on the cannon and spam the first ability, until the second ability is available, then rinse and repeat.

**Comment by Echinaceax** [comment]
yes @flubbacrud  i know, i had the same issue on my pala when i did the questline, just forgot to update it here

**Comment by Echinaceax** [comment]
Updated info from @gorgar19 #5317

After completing this via the BQ command player phases again and cant see the skulls for the next quest reported here #4593

@mfranca0009 
full questline will need to be reported to ensure phasing is fixed correct

I will make the report when i can get on PTR
